Updating some CV things (#14)
cgranade/cgranade.github.io: _includes/docs/cv.md

  
## Recent Work History ##

> - Owner and Principle at Dual Space Solutions LLC. May 2023 to present. 

Reviewer: ```suggestion
- Owner and Principal, Dual Space Solutions LLC. May 2023 to present. 
```

> - Senior Research Developer, Microsoft. July 2017 to October 2022.

Reviewer: ```suggestion
- Senior Research Development Engineer, Microsoft. July 2017 to October 2022.
```

  > [https://doi.org/10.3389/fphy.2022.940293](https://doi.org/10.3389/fphy.2022.940293)

#### 2021 ####

- *Learning models of quantum systems from experiments*, Gentile, A.A., Flynn, B., Knauer, S., Wiebe, N., Paesani, S., **Cassandra E. Granade**, Rarity, J.G., Santagati, R. and Laing, A., 2021.  Nature Physics, 17(7), pp.837-843.
  > [https://doi.org/10.1038/s41567-021-01201-7](https://doi.org/10.1038/s41567-021-01201-7)

#### 2020 ####

- *Toward quantum computing for high-energy excited states in molecular systems: quantum phase estimations of core-level states*, Bauman, N.P., Liu, H., Bylaska, E.J., Krishnamoorthy, S., Low, G.H., **Cassandra E. Granade**, Wiebe, N., Baker, N.A., Peng, B., Roetteler, M. and Troyer, M. Journal of Chemical Theory and Computation, 17(1), pp.201-210.
  > [https://doi.org/10.1021/acs.jctc.0c00909](https://doi.org/10.1021/acs.jctc.0c00909)

- *Operational, gauge-free quantum tomography*, Di Matteo, O., Gamble, J., **Cassandra E. Granade**, Rudinger, K. and Wiebe, N. Quantum, 4, p.364. 
  > [https://doi.org/10.22331/q-2020-11-17-364](https://doi.org/10.22331/q-2020-11-17-364)

#### 2019 ####

> - *Q# and NWChem: tools for scalable quantum chemistry on quantum computers*, Low, G.H., Bauman, N.P., **Cassandra E. Granade**, Peng, B., Wiebe, N., Bylaska, E.J., Wecker, D., Krishnamoorthy, S., Roetteler, M., Kowalski, K. and Troyer, M.

Reviewer: ```suggestion
- *Q# and NWChem: Tools for scalable quantum chemistry on quantum computers*, Low, G.H., Bauman, N.P., **Cassandra E. Granade**, Peng, B., Wiebe, N., Bylaska, E.J., Wecker, D., Krishnamoorthy, S., Roetteler, M., Kowalski, K. and Troyer, M.
```

  > [arXiv:1904.01131](https://doi.org/10.48550/arXiv.1904.01131)

- *Bayesian ACRONYM Tuning*, Gamble, J., **Cassandra E. Granade** and Wiebe, N.
  > [arXiv:1902.05940](https://doi.org/10.48550/arXiv.1902.05940)

- *Magnetic-field learning using a single electronic spin in diamond with one-photon readout at room temperature*, Santagati, R., Gentile, A.A., Knauer, S., Schmitt, S., Paesani, S., **Cassandra E. Granade**, Wiebe, N., Osterkamp, C., McGuinness, L.P., Wang, J. and Thompson, M.G. Physical Review X, 9(2), p.021019.
  > [https://doi.org/10.1103/PhysRevX.9.021019](https://doi.org/10.1103/PhysRevX.9.021019)

#### 2018 ####

- *Statistical inference with quantum measurements: methodologies for nitrogen vacancy centers in diamond*, Hincks, I., **Cassandra E. Granade** and Cory, D.G. New Journal of Physics, 20(1), p.013022.
  > [https://doi.org/10.1103/PhysRevApplied.8.064004](https://doi.org/10.1103/PhysRevApplied.8.064004)

> - *Q# enabling scalable quantum computing and development with a high-level dsl*, Svore, K., Geller, A., Troyer, M., Azariah, J.,**Cassandra E. Granade**, Heim, B., Kliuchnikov, V., Mykhailova, M., Paz, A. and Roetteler, M., 2018, February. In Proceedings of the real world domain specific languages workshop 2018 (pp. 1-10).

Reviewer: ```suggestion
- *Q#: Enabling scalable quantum computing and development with a high-level dsl*, Svore, K., Geller, A., Troyer, M., Azariah, J.,**Cassandra E. Granade**, Heim, B., Kliuchnikov, V., Mykhailova, M., Paz, A. and Roetteler, M., 2018, February. In Proceedings of the real world domain specific languages workshop 2018 (pp. 1-10).
```
